user_data.sh

```diff
@@ -16,6 +16,19 @@ setfacl -Rdm other:0 /var/log/bastion
 # Make OpenSSH execute a custom script on logins
 echo -e "\\nForceCommand /usr/bin/bastion/shell" >> /etc/ssh/sshd_config
 
+sed -i 's/MaxAuthTries\ [0-9]/MaxAuthTries 5/' /etc/ssh/sshd_config
+
+# Deny interactive shell to some users (tunnel-only)
+cat >> /etc/ssh/sshd_config << 'EOF'
```

Comment:
This all looks sane. Though we do have some compliance stuff that will need to be addressed and some existing configs/opinions regarding the sshd config (in the sysadmin repo). These are things like the login banner and session timeout.

Reply:
Not an issue for my current non-interactive use case, but I'll address in another PR if we use this module to create human-access bastion.